tests/integration/sugar/test_account.py

```diff
@@ -0,0 +1,45 @@
+from unittest import TestCase
```

Comment:
i don't think we need a `sugar` dir for integration tests. i get it (and you get it), but it doesn't match with a source dir. i can see either having a 1:1 dir struct like we have for unit tests, or a _less_ granular struct for integration tests. but i'm open to ideas. we can research this

Reply:
i will say for now, because we have deadlines, lets _not_ do the sugar dir. i might be wrong.

Reply:
This was created to make it easier to run all these tests at once - if they're in a folder that has an __init__, then you can run them all with one command. Happy to take it out though.